integrationtests.py

```diff
@@ -578,25 +579,33 @@ def testSampleproject(self):
 
 class TestHeaderChange(unittest.TestCase):
     def _clean(self):
+        # It seems that subprocess.check_output() occansionally returns before
+        # windows fully releases the respective executable.
+        # This pause prevents failing tests because of missing permissions to remove the file.
+        time.sleep(.1)
+
```

Comment:
This is an improvement, but it suffers from two downsides: .1 might still not be enough for some cases, so the test remains flaky (though less so), and now the tests *always* waits .1 no matter whether it's necessary or not.

A common alternative (retrying and ignoring the 'Access denied' error) would be more code though, so for now let's just go for this and see if/when it breaks.